packages/beacon-node/src/api/impl/validator/index.ts

```diff
@@ -399,25 +399,15 @@ export function getValidatorApi(
   async function produceBuilderBlindedBlock(
     slot: Slot,
     randaoReveal: BLSSignature,
-    graffiti?: string,
+    graffiti: Buffer,
```

Comment:
I think we should avoid using `Buffer` type for new code if doesn't require it

```suggestion
    graffiti: Uint8Array,
```

Reply:
or even better, use `Bytes32` like here https://github.com/ChainSafe/lodestar/blob/f289ef591653255821adefff9aff9aae82ba8692/packages/beacon-node/src/chain/produceBlock/produceBlockBody.ts#L77